lib/Backend/Lower.cpp

```diff
@@ -597,7 +597,7 @@ Lowerer::LowerRange(IR::Instr *instrStart, IR::Instr *instrEnd, bool defaultDoFa
             break;
 
         case Js::OpCode::InlineMathFloor:
-#if _M_X64
+#ifdef ASMJS_PLAT
```

Comment:
ASMJS_PLAT isn't the appropriate #define here as this is platform specific code.  Should be (x86 || x64) && ASMJS_PLAT.